UPGRADE.asciidoc

```diff
@@ -1,21 +1,17 @@
 = PacketFence Upgrade Guide
-:encoding: UTF-8
-:lang: en
-:doctype: book
-:toc: left
-
-// to display images directly on GitHub
-ifdef::env-github[]
-:imagesdir: images
-:extra_path: docs/
-endif::[]
-:extra_path:
 ////
 
     This file is part of the PacketFence project.
 
-////
+    See docs/includes/global-attributes.asciidoc for authors, copyright and
+    license information.
 
+////
+// Fix includes on GitHub
+:extra_path:
+ifdef::env-github[]
+:extra_path: docs/
+endif::[]
```

Comment:
If think we can remove that part and all `github[]` references in files under docs/*/*.asciidoc

Reply:
It's still relevant.